src/main/java/de/fearnixx/jeak/service/token/RandomString.java

```diff
@@ -0,0 +1,65 @@
+package de.fearnixx.jeak.service.token;
+
+import java.security.SecureRandom;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.Random;
+
+/*
+taken from: https://stackoverflow.com/questions/41107/how-to-generate-a-random-alpha-numeric-string/41156#41156
+and tweaked a little bit.
+ */
+public class RandomString {
+
+    public static final String upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+    public static final String lower = upper.toLowerCase(Locale.ROOT);
+
+    public static final String digits = "0123456789";
+
+    public static final String alphanum = upper + lower + digits;
+
+    private final Random random;
+
+    private final char[] symbols;
+
+    private final char[] buf;
+
+    public RandomString(int length, Random random, String symbols) {
+        if (length < 1) throw new IllegalArgumentException();
+        if (symbols.length() < 2) throw new IllegalArgumentException();
```

Comment:
I know this is very picky, but please always provide messages when throwing exceptions :slightly_smiling_face: